source/draw/Color.ooc

```diff
@@ -24,6 +24,9 @@ ColorMonochrome: cover {
 	toRgba: func -> ColorRgba { this toYuv() toRgba() }
 	equals: func (other: This) -> Bool { this y == other y }
 	mix: static func (a, b: This, ratio: Float) -> This { This new(Float mix(a y, b y, ratio) as Byte) }
+	mix: static func ~bilinear (ratio: FloatPoint2D, topLeft, topRight, bottomLeft, bottomRight: This) -> This {
+		ColorMonochrome new((Float mix(Float mix(topLeft y as Float, topRight y as Float, ratio x), Float mix(bottomLeft y as Float, bottomRight y as Float, ratio x), ratio y) round()) clamp(0.0f, 255.0f) as Byte)
```

Comment:
round is not really needed since the byte is unsigned. Adding 0.5 is much faster.